liquidator/index.js

```diff
@@ -61,33 +62,39 @@ async function run(
 
     const getTime = () => Math.round(new Date().getTime() / 1000);
 
-    // Setup web3 accounts, account instance and pricefeed for EMP.
-    const [accounts, emp, voting, priceFeed] = await Promise.all([
+    // Load unlocked web3 accounts, get the networkId and set up price feed.
+    const [accounts, networkId, priceFeed] = await Promise.all([
       web3.eth.getAccounts(),
-      ExpiringMultiParty.at(empAddress),
-      Voting.deployed(),
+      web3.eth.net.getId(),
       createReferencePriceFeedForEmp(logger, web3, new Networker(logger), getTime, empAddress, priceFeedConfig)
     ]);
 
     if (!priceFeed) {
       throw new Error("Price feed config is invalid");
     }
 
+    // Setup contract instances.
+    const voting = new web3.eth.Contract(getAbi("Voting"), getAddress("Voting", networkId));
```

Comment:
When originally writing the core entrypoint, I wanted to keep it as simple as possible to avoid making assumptions about use. Curious to hear your thoughts on adding a convenience function to it that basically does this for you:
```js
const [voting, emp] = await getWeb3Instance(web3, ["Voting", "ExpiringMultiParty"]);
```

Or other APIs that you think make sense?

Note: since we're only using this in this place, we can add this later. This shouldn't block this PR or anything.

Reply:
I think that that's pretty overkill for now. We could add this kind of wrapper later down the line but for now, I think it's simplest to let the implementer initiate the web3 instance. I like that the functions give me the address and ABI and then I can do the rest. Feels like you have more control for different use cases IMHO.

Reply:
Cool cool, makes sense!